RecoCTPPS/Configuration/test/raw_data_test.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TotemStandaloneRawDataTest")
+
+# minimum of logs
+process.MessageLogger = cms.Service("MessageLogger",
+    statistics = cms.untracked.vstring(),
+    destinations = cms.untracked.vstring('cout'),
+    cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('WARNING')
+    )
+)
+
+# raw data source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/afs/cern.ch/user/j/jkaspar/public/run273062_ls0001-2_stream.root')
```

Comment:
express doesn't sound like the best choice.
Use RAW data from a physics dataset seems more appropriate.
If file goes away from disk, it can be requested back using a PhedEx request
a longer term solution is a file in /store/group or /store/user